dispatcher/http.go

```diff
@@ -0,0 +1,75 @@
+package dispatcher
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/whosonfirst/go-webhookd/v3"
+)
+
+func init() {
+
+	ctx := context.Background()
+	err := RegisterDispatcher(ctx, "http", NewHTTPDispatcher)
```

Comment:
It's worth adding a second call to `RegisterDispatcher` to register "https" URIs since some or all of the dispatcher specifics should be encoded in the URI string passed to `NewHTTPDispatcher`.

Reply:
Added